src/components/markdown/ProblemsList/ProblemStatusCheckbox.tsx

```diff
@@ -9,21 +9,28 @@ import UserDataContext from '../../../context/UserDataContext/UserDataContext';
 import Tippy from '@tippyjs/react';
 import 'tippy.js/themes/light.css';
 import styled from 'styled-components';
-
+import feedbackModal from '../../feedbackModal';
 import MarkdownLayoutContext from '../../../context/MarkdownLayoutContext';
 import ConfettiContext from '../../../context/ConfettiContext';
+import Transition from '../../Transition';
+import { logError } from 'gatsby/dist/state-machines/develop/actions';
+import * as firebase from 'firebase';
+import FirebaseContext from '../../../context/FirebaseContext';
 
 const StyledTippy = styled(Tippy)`
   .tippy-content {
     padding: 0;
   }
 `;
 
-const ProgressDropdown = ({ onProgressSelected, currentProgress }) => {
-  const [activeProgress, setActiveProgress] = useState<ProblemProgress>(
-    currentProgress
-  );
-
+ 
+const ProgressDropdown = ({
+  onProgressSelected,
+  currentProgress,
+  openModal,
```

Comment:
what is `openModal` doing?

Reply:
@thecodingwizard It's either useless or it's essential to the feedback system so I wouldn't change that.

Reply:
figure it out!!! Somebody has to maintain this code LOL

Reply:
fine fine...

Reply:
Yeah its not used